packages/esm-service-queues-app/src/queue-entry-table-components/actions-menu.component.tsx

```diff
@@ -21,35 +20,33 @@ const ActionsMenu: React.FC<ActionsMenuProps> = ({ queueEntry, closeModal }) =>
   }, [queueEntry]);
 
   return (
-    <Layer>
```

Comment:
Removed `<Layer>`. It caused some CSS issues in `ActiveVisitRowActionsCell` and didn't seem to have noticeable impact with the old UI